### PR TITLE
RFC git commit messages

### PR DESCRIPTION
**Pull request by yngvark**
## Description
A RFC for github comments.

## Motivation and Context

See the PR changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have updated the release notes (for the next release).

### CODE REVIEW

File: docs/rfc/0002-git-commit-messages.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+<!--
+    Short summary of the decision
+
+    Example:
+    We've decided to go for AlphaGov's system as explained [here](https://github.com/alphagov/govuk-aws/blob/24d1ea513e58ee938043d71d09815a51229067bf/docs/architecture/decisions/0001-record-architecture-decisions.md)
+-->
+
+Git commit messages found on the **master** branch should in **most cases** link to an issue, giving context to
+the commit. The issue can be a `#`followed by Github issue number or `KM-` followed by a Trello card number.
```
Comment (paulbes) [inline docs/rfc/0002-git-commit-messages.md:52]: Could we change the `KM` section to `KM{0..9}-`, please!

File: docs/rfc/0002-git-commit-messages.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+```text
+feat(ux): Add userdocs for okctl create cluster
+
+#123
+```
+
+Trello examples:
+
+```text
+KM-160: Add userdocs for okctl create cluster
```
Comment (paulbes) [inline docs/rfc/0002-git-commit-messages.md:77]: `KM160` 🙏🏽

**Review by paulbes — COMMENTED** [review]
Love it ❤️

**Review by deifyed — APPROVED** [review]

**Review by ivaruf — APPROVED** [review]
Very nice :)

File: docs/rfc/0002-git-commit-messages.md (hunk 1 above)
Comment (yngvark) [reply docs/rfc/0002-git-commit-messages.md]: Done, good idea!

**Pull request merged**